infrastructure/quick-deploy/gcp/all-in-one/chaos-mesh.tf

```diff
@@ -0,0 +1,34 @@
+# Chaos Mesh
+module "chaos-mesh" {
+  count     = var.chaos-mesh.enable ? 1 : 0
+  source    = "../../../modules/k8s/chaos-mesh/"
+  namespace = var.chaos-mesh.namespace
+  docker_image = {
+    chaosmesh = {
+      image = var.chaos-mesh.chaosmesh_image_name
+      tag   = try(coalesce(var.chaos-mesh.chaosmesh_image_tag), local.default_tags[var.chaos-mesh.chaosmesh_image_name])
+    }
+    chaosdaemon = {
+      image = var.chaos-mesh.chaosdaemon_image_name
+      tag   = try(coalesce(var.chaos-mesh.chaosdaemon_image_tag), local.default_tags[var.chaos-mesh.chaosdaemon_image_name])
+    }
+    chaosdashboard = {
+      image = var.chaos-mesh.chaosdashboard_image_name
+      tag   = try(coalesce(var.chaos-mesh.chaosdashboard_image_tag), local.default_tags[var.chaos-mesh.chaosdashboard_image_name])
+    }
+  }
+  helm_chart_repository = try(coalesce(var.chaos-mesh.helm_chart_repository), var.helm_charts.chaos-mesh.repository)
+  helm_chart_version    = try(coalesce(var.chaos-mesh.helm_chart_verison), var.helm_charts.chaos-mesh.version)
+  service_type          = var.chaos-mesh.service_type
+}
+
+data "external" "get_chaos_dashboard_port" {
+  program    = ["sh", "-c", "kubectl get svc chaos-dashboard -n chaos-mesh -o jsonpath='{\"{\"}\"nodeport\": \"{.spec.ports[0].nodePort}\"}'"]
+  depends_on = [module.chaos-mesh]
+}
+
+data "external" "get_chaos_dashboard_ip" {
+  program    = ["sh", "-c", "kubectl get svc chaos-dashboard -n chaos-mesh -o jsonpath='{\"{\"}\"ip\": \"{.status.loadBalancer.ingress[0].ip}\"}'"]
+  depends_on = [module.chaos-mesh]
+}
```

Comment:
This should most likely be in the chaos-mesh module.

Reply:
done